plugins/tiddlywiki/multiwikiserver/modules/store/sql-tiddler-store.js

```diff
@@ -151,8 +151,9 @@ SqlTiddlerStore.prototype.processIncomingTiddler = function(tiddlerFields) {
 	const attachmentsEnabled = this.adminWiki.getTiddlerText("$:/config/MultiWikiServer/EnableAttachments","yes") === "yes";
 	const contentTypeInfo = $tw.config.contentTypeInfo[tiddlerFields.type || "text/vnd.tiddlywiki"],
 		isBinary = !!contentTypeInfo && contentTypeInfo.encoding === "base64";
-	if(attachmentsEnabled && isBinary && tiddlerFields.text && tiddlerFields.text.length > attachmentSizeLimit) {
-		const attachment_blob = this.attachmentStore.saveAttachment({
+
+	if (attachmentsEnabled && isBinary && ((tiddlerFields.text && tiddlerFields.text.length > attachmentSizeLimit) || (!!tiddlerFields?.attachment_blob && tiddlerFields?.attachment_blob?.length < attachmentSizeLimit))) {
```

Comment:
I am afraid that our coding standards are somewhat old fashioned, and are not currently enforced by tooling. There should not be whitespace between `if` and the following parenthesis `(`.